A32-xds-circuit-breaking.md

```diff
@@ -53,13 +53,23 @@ circuit breakers. Users can effectively turn circuit breaking off by setting
 limits to very high values, for example, to 
 `std::numeric_limits<uint32_t>::max()`.
 
+## Related Proposals: 
+
+This proposal builds on the earlier xDS work described in
+- [A27: xDS-Based Global Load Balancing](https://github.com/grpc/proposal/blob/master/A27-xds-global-load-balancing.md)
+
+Related design changes made after this proposal in
+- [A37: xDS Aggregate and Logical DNS Clusters](https://github.com/grpc/proposal/pull/216)
```

Comment:
I think the change here can be explained without actually referencing A37 at all.  I think the only thing we really need to say is that we want to be consistent with Envoy, where the circuit breaking counts are tracked on a per-process basis, not a per-client basis.  So we want the counts to be tracked the same way in gRPC.

Reply:
Deleted. So I am only adding a single paragraph to mention circuit breaking is a stateful feature, so the state should be shared for channels in the entire process. See the added section below.